src/computed-style-property-map.js

```diff
@@ -78,15 +103,28 @@
 
     switch (property) {
```

Comment:
Sounds like a good idea for now, might need to restructure later on.  It is getting quite unwieldy, I agree.